### DISCUSSION TITLE
wss connection issue on blockscout frontend

### DISCUSSION THREAD

**Discussion by Neoblock8** [blockscout/blockscout#9830] [Q&A]
Hi,
 Running in docker. Connecting blockscout frontend with backend. It shows webscocket error: "wss://example.net/socket/v2/websocket?vsn=2.0.0". How to overcome this issue

**Comment by GimluCom** [comment]
Is it working without SSL? Are you using the default nginx template?

**Comment by Karthigasrini** [comment]
> Is it working without SSL? Are you using the default nginx template?

How to check it is working with ssl. Not using default nginx template

**Reply by GimluCom** [reply]
SSL is working if you can access your instance at https://localhost

**Comment by ekawibisono** [comment]
hi @Karthigasrini In your problem has solved ? or not ?
